README.md

```diff
@@ -26,14 +26,15 @@ The Oculus Quest uses the new XR plugin system, since the project was supposed t
 In-Editor testing: The network panel is set to show extra entries in Editor mode - this proved useful in UI design, and is a useful feature (using conditional compilation) should this be required elsewhere. Much of the `SceneLoader` component uses conditional compilation to set up cameras and UI for different build targets.
 
 ### Unity Cloud Build
-Unity Cloud Build is currently used from `Master` branch for most builds. You can build standalone desktop builds at any time on either Mac or Windows if required. This is not much use beyond those build targets, since it cannot build for Quest or Hololens.
+As of July, 2021 Unity Cloud Build are no longer running. Previously Cloud Builds were created from the `master` branch for Mac and Windows. Instead builds are done locally and then shared via S3. Furthermore, Cloud Builds were not much use beyond Mac and Windows build targets, since Unity Cloud Builds do not support the Quest or Hololens.
```

Comment:
```suggestion
As of July 2021 Unity Cloud Build is no longer running. Previously Cloud Builds were created from the `master` branch for Mac and Windows. Instead builds are done locally and then shared via S3. Furthermore, Cloud Builds were not much use beyond Mac and Windows build targets, since Unity Cloud Builds do not support the Quest or Hololens.
```
or perhaps "Unity Cloud Builds are no longer running."